src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -336,6 +336,11 @@ public static class ConfigurationKeys
         /// </summary>
         public const string JaegerExporterAgentPort = "OTEL_EXPORTER_JAEGER_AGENT_PORT";
 
+        /// <summary>
+        /// Configuration key for specifing vendor plugin path
+        /// </summary>
+        public const string VendorPluginName = "OTEL_VENDOR_PLUGIN";
```

Comment:
Open question. Is it desired to allow multiple plugins? Especially that https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/117 tells about "plugins". I think it would be safer to think about multiple plugins even though it complicates the design. Especially taking into consideration that the plugins should EXTEND and not REPLACE the core functionality (at least in my understanding). I think that all extension points should work like enumerators/decorators/middlewares.